pkg/web/api/v1/user.go

```diff
@@ -24,6 +25,11 @@ type UserList struct {
 	Users []*User    `json:"users"`
 }
 
+type UserPassword struct {
+	Password  string `json:"password"`
+	SendEmail bool   `json:"send_email"`
```

Comment:
`SendEmail` currently doesn't do anything, but in the future I'm thinking that if this is true, Envoy could send the user an email with a password change notice.